ui/app/common/pnc-client/resources/BuildResource.js

```diff
@@ -259,21 +240,6 @@
         return canonicalName(this);
       };
 
-
-      function buildLogUrl(build) {
```

Comment:
Actually I can see these methods (`$buildLogUrl` and also `$alignLogUrl`) are still used in the UI, why did you delete them?

See for example: `ui/app/builds/detail/alignment-log/pnc-build-detail-alignment-log-page.html`

Reply:
supposedly we should no longer use `/logs/build` and `/logs/align` endpoints because they are going to be removed, I will revert the deleted part for now and discuss with Jan to see how we could handle the download part with new Bifrost endpoints.